README.md

```diff
@@ -1,70 +1,39 @@
-# Getting Started with Create React App
+# 2022 August - Hawk's Cohort - React Invoice App
+
+## 1. About
+### Content
+Project 4 relating to module 5 of iO Full-Stack Web Development.
+
+### Objective
+Group project with the objective of build an Invoice Management Application to test knowledge acquired 
+during module 5 of iO Academy Full-Stack curriculum:
++ Agile work environment best practices in a SCRUM methodology
++ Git / Github workflow and best practices as a developer team
++ Component Driven Development using the React.js Library
++ CSS frameworks  using Bootstrap 5.2.2
+
+### Technologies
++ React.js - https://reactjs.org/
++ Bootstrap - https://getbootstrap.com/
++ Git - https://github.com/com
+
+## 2. Installation
+Follow the next steps to install locally:
++ ### Install Git
+  Using the following the instructions for the corresponding OS - https://github.com/git-guides/install-git
++ ### Installing the Data Base
```

Comment:
Go with "Database" - most people are ok with the one word.

Reply:
done